src/catalog/src/durable/impls/persist.rs

```diff
@@ -578,6 +571,10 @@ pub struct PersistCatalogState {
 }
 
 impl PersistCatalogState {
+    pub fn desc() -> RelationDesc {
+        RelationDesc::empty().with_column("kind", ScalarType::Jsonb.nullable(false))
```

Comment:
It was easier to prototype without pulling out the tag as a separate column, but this any particular opinion on my part. I wonder if we can find someone who has opinions on what this json should look like